components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -242,7 +260,11 @@ export default function () {
                     {((!isDetecting && isEditorDisabled) || (prebuildInstance?.status.phase === "stopped" && !prebuildInstance?.status.conditions.failed))
                         ? <a className="my-auto" href={`/#${project?.cloneUrl}`}><button className="secondary">New Workspace</button></a>
                         : <button disabled={true} className="secondary">New Workspace</button>}
-                    <button disabled={isDetecting || (prebuildWasTriggered && prebuildInstance?.status.phase !== "stopped")} onClick={buildProject}>Run Prebuild</button>
+                    {(prebuildWasTriggered && prebuildInstance?.status.phase !== "stopped")
+                        ? <button className="danger flex items-center space-x-2" disabled={prebuildWasCancelled || (prebuildInstance?.status.phase !== "initializing" && prebuildInstance?.status.phase !== "running")} onClick={cancelPrebuild}>
+                            <span>Cancel Prebuild</span>
```

Comment:
**issue**: This is most probably out of the scope of these changes but when canceling a prebuild from the project configurator, the prebuild status on the left side of the configurator falls back to `READY`. Could we introduce a `CANCELED` status and use this as a status here? Pasting below an early draft featuring the `CANCELED` status. Is this technically feasible and does this sound expected?

<img width="931" alt="Screenshot 2021-10-04 at 7 01 28 PM" src="https://user-images.githubusercontent.com/120486/135884824-3cfc6b8d-79c7-4bdb-8f19-aa66d256e946.png">